docs/index.rst

```diff
@@ -0,0 +1,62 @@
+:relatedlinks: [Diátaxis](https://diataxis.fr/)
+
+.. _home:
+
+Charmed MLflow Documentation
+============================
+
+Charmed MLflow is a platform for managing the end-to-end machine learning lifecycle.
+
+It provides tools for tracking experiments, packaging code into reproducible runs, and sharing and deploying models. It integrates with popular machine learning frameworks.
+
+It addresses a number of challenges facing machine learning teams: collaboration, reproducibility, maintenance, organisation and scaling.
```

Comment:
```suggestion
It addresses a number of challenges that machine learning teams often face: collaboration, reproducibility, maintenance, organisation and scaling.
```

Reply:
Thanks, it was a bit off. I reworded it differently.